ete3/ncbi_taxonomy/ncbiquery.py

```diff
@@ -355,7 +355,12 @@ def get_descendant_taxa(self, parent, intermediate_nodes=False, rank_limit=None,
                 break
             
         if not found:
-            raise ValueError("taxid not found:%s" %taxid)
```

Comment:
sorry, this is conceptually wrong. you should return the converted taxid, but its descendants. Please this part as in the original version, and use the suggested fix a few lines before